### PR TITLE
Created vav.cpp

### LINKED ISSUE #4: I'm not comfortable here
This is a sample issue.

### PR DESCRIPTION
**Pull request by ghost-vaibhav**
_A simple program which adds two numbers and displays it._

### CODE REVIEW

**Comment by GhostVaibhav** [comment]
I will merge it in a short while, thanks for concerning us.

**Comment by ghost-vaibhav** [comment]
Ok, thanks for taking this into consideration

File: vav.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include<iostream>
+using namespace std;
+#define sum(x,y) x+y
```
Comment (GhostVaibhav) [inline vav.cpp:3]: I don't like this, please modify this and convert to a function

Comment (GhostVaibhav) [reply vav.cpp]: Definitely, I will do that

**Review by ghost-vaibhav — COMMENTED** [review]
I will definitely change this ASAP, but please merge this and I will kick on with the new work.

File: vav.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include<iostream>
+using namespace std;
+int sum(int x, int y)
+{
+    return x+y;   
+}
```
Comment (ghost-vaibhav) [inline vav.cpp:6]: Yes, this is good, now you can merge it easily

**Comment by GhostVaibhav** [comment]
Yes, it's good, let's merge it

**Review by GhostVaibhav — APPROVED** [review]

**Review by GhostVaibhav — APPROVED** [review]
I will appprove

**Pull request merged**